crawler/code/crawling.py

```diff
@@ -141,7 +141,7 @@ def parse_links(self, response):
                 text = yield from response.text()
 
                 # Replace href with (?:href|src) to follow image links.
-                urls = set(re.findall(r'''(?i)href=["']?([^\s"'<>]+)''',
+                urls = set(re.findall(r'''(?i)href=["']([^\s"'<>]+)''',
```

Comment:
Yes! This is the commit: https://github.com/ajdavis/500lines/commit/470cba87faa56f9eb941f14f9b1a1507888f7f6c.

The new comic at http://xkcd.com/1572/ has some Javascript like:
```
document.getElementById("id").href=["http://example.com"];
```
... so that reveals a bug in this regex, which thinks the URL is "[". So now I removed the "?" to require an opening quote in the URL.

(Really I should just use BeautifulSoup....)